plugin.py

```diff
@@ -417,6 +419,11 @@ def onMQTTPublish(self, topic, rawmessage):
                 object_id = topiclist[discoverytopiclen+2]
                 action = topiclist[discoverytopiclen+3]
 
+              # Sensor support
```

Comment:
Briefly explain what is happening here instead of just "Sensor support"

Reply:
BLE sensors message has extra node_id field, have to correct for it